### ISSUE TITLE
ChromeOS Favicons are not working.

### ISSUE THREAD

**Issue by AfterNetwork** [AfterNetwork/ConnectOneTwo#14]
Letting you know.

**Comment by jbrow78** [comment]
I know. I am working on it.
BTW, plz vote on new J Tech logo

**Comment by smileycreations15** [comment]
@jbrow78 u really need to advertise J tech??!??!?!
GIVE ME OFFICIAL DOCUMENTS OF JTECH TO PROVE IT EXISTS.

**Comment by jbrow78** [comment]
I need someone to approve pull request

**Comment by smileycreations15** [comment]
@jbrow78 WHAT ARE YOU DOING BY DELETING MY COMMENTS! 
YOU ARE PURE STUPID? PLEASE DO NOT DELETE MY COMMENTS JUST TO HIDE EVIDENCE
PROVE J TECH IS REAL AND STOP ADVERTISING IT

**Comment by smileycreations15** [comment]
@jbrow78 WHAT ARE YOU DOING BY DELETING MY COMMENTS!
PLEASE DO NOT DELETE MY COMMENTS JUST TO HIDE EVIDENCE
PROVE J TECH IS REAL AND STOP ADVERTISING IT

**Comment by jbrow78** [comment]
> @jbrow78 WHAT ARE YOU DOING BY DELETING MY COMMENTS!
> PLEASE DO NOT DELETE MY COMMENTS JUST TO HIDE EVIDENCE
> PROVE J TECH IS REAL AND STOP ADVERTISING IT

This is not the place to do this. This is not just one of your projects where you can do anything you want. AFN and I are actually trying to make this not just trying to mess around.

**Comment by smileycreations15** [comment]
i have the right to talk.

**Comment by smileycreations15** [comment]
@jbrow78

**Comment by smileycreations15** [comment]
i have the right to talk.

**Comment by jbrow78** [comment]
> i have the right to talk.

Yes, but be relevant, or else I will have After remove you from the project.
And plz approve my pull request.

**Comment by AfterNetwork** [comment]
The favicons are still broken.

**Comment by AfterNetwork** [comment]
> i have the right to talk.

I can decide to kick you from the project at any time. (also smiley you haven't done any work on this project, so maybe start working on it)

**Comment by jbrow78** [comment]
Ok. So i think i know how to fix it.

**Comment by AfterNetwork** [comment]
> Ok. So i think i know how to fix it.

ok then fking fix it
